web/geo/validate.js

```diff
@@ -132,21 +132,26 @@ function zeroOverlaps(fc, { MIN_AREA_INTERSECTION }) {
   // add bbox to features to increase speed
   const features = getPolygons(fc).features.map((ft) => ({ ft, bbox: bboxPolygon(bbox(ft)) }));
 
+  const countriesToIgnore = fc.features.filter((ft) => ft.properties.isCombined).map((ft) => ft.properties.countryKey);
+
   const overlaps = features
     .filter(
       (ft1, idx1) =>
         features.filter((ft2, idx2) => {
           if (idx1 !== idx2 && intersect(ft1.bbox, ft2.bbox)) {
             const intersection = intersect(ft1.ft, ft2.ft);
             if (intersection && area(intersection) > MIN_AREA_INTERSECTION) {
+              if (countriesToIgnore.includes(ft1.ft.properties.countryKey)) {
+                return false;
+              }
```

Comment:
Shouldn't we have this conditional higher up? I.e is there a reason that we compute stuff related to the countriesToIgnore, or could we just filter them off in the beginning?